_kbarticles/2023-08-07-advantages-of-multiple-application-instances.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Scaling your cloud.gov applications"
+date: August 7, 2023
+excerpt: How and why to scale your cloud.gov applications by adding more application instances
+---
+
+## Importance of multiple application instances
+
+One of the benefits of Cloud.gov is the ability to [deploy applications with a simple command](https://cloud.gov/docs/deployment/deployment/#how-deployment-works). 
+
+Once an application is up and available, resiliency of that application becomes a consideration and as such running multiple application instances is paramount. Running multiple instances of an application increases the resilience and availability of the application.
+
+Multiple application instances allow customer traffic to continue being served if there are any issues with any individual application instance such as if an underlying platform component is being upgraded. Additionally having multiple application instances provides decreased chance of downtime. 
+
+Running multiple application instances allows your instances to be [scheduled across availability zones](https://docs.aws.amazon.com/whitepapers/latest/real-time-communication-on-aws/use-multiple-availability-zones.html). The increased number of instances also assists with load-balancing user requests to your application (Please note: you should set up your instances to share session data when using multiple application instances).
```

Comment:
While the diego cell VMs that host applications are in multiple availability zones, the diego cells for your instances are chosen randomly. So there is no guarantee that more application instances leads to more availability zones